Dockerfile

```diff
@@ -14,6 +14,9 @@ RUN apt-get update \
     && curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
     && echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list
 
+
+ENV DEBIAN_FRONTEND=noninteractive
+
```

Comment:
What prompt issue is more exactly fixed by setting `DEBIAN_FRONTEND`? I'm fine with the change, but I thought executing `apt-get install` command with the `-y` option was preventing a confirmation prompt already?